input/input.c

```diff
@@ -275,11 +275,20 @@ static struct mp_cmd *queue_peek_tail(struct cmd_queue *queue)
     return cur;
 }
 
+static int queue_cmd(struct input_ctx *ictx, mp_cmd_t *cmd)
+{
+    if (cmd) {
+        queue_add_tail(&ictx->cmd_queue, cmd);
+        mp_input_wakeup(ictx);
+    }
+    return 1;
```

Comment:
I know it was like that, but this really bothers me. Could we `return 0` if nothing happens i.e. cmd is null? Or remove return value completely from static function if it is useless.

I would prefer something like the following. But whatever is fine for me as long as return value is meaningful or not removed.
``` C
static void queue_cmd(struct input_ctx *ictx, mp_cmd_t *cmd)
{
    if (!cmd)
        return;
    queue_add_tail(&ictx->cmd_queue, cmd);
    mp_input_wakeup(ictx);
}

int mp_input_queue_cmd(struct input_ctx *ictx, mp_cmd_t *cmd)
{
    if (!cmd)
        return 0;
    input_lock(ictx);
    queue_cmd(ictx, cmd);
    input_unlock(ictx);
    return 1;
}
```